### PR TITLE
ensure that nobrief output is handled correctly

### PR DESCRIPTION
**Pull request by mdehoon**
Ensure that EMBOSS output generated using the `-nobrief` qualifier is parsed correctly.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit`` locally,
and understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/Align/emboss.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 # package.
 """Bio.Align support for "emboss" alignment output from EMBOSS tools.
 
-This module contains a parser for the EMBOSS pairs/simple file format, for
-example from the alignret, water and needle tools.
```
Comment (peterjc) [inline Bio/Align/emboss.py:10]: Does ``alignret`` not exist in EMBOSS anymore or something?

Comment (mdehoon) [reply Bio/Align/emboss.py:10]: I believe so, yes. I didn't find it in the current EMBOSS distribution, and google only finds it in biopython.

**Comment by mdehoon** [comment]
@peterjc  Any objections?

**Review by peterjc — APPROVED** [review]

**Comment by mdehoon** [comment]
@peterjc  Thank you.

**Pull request merged**